publish.go

```diff
@@ -184,17 +194,30 @@ func (publisher *Publisher) Publish(
 	}
 
 	for _, routingKey := range routingKeys {
+		var message = amqp.Publishing{}
+		message.ContentType = options.ContentType
+		message.DeliveryMode = options.DeliveryMode
+		// Message Body
+		message.Body = data
+
+		// If no header options, don't add.
+		if len(options.Headers) > 0 {
+			message.Headers = tableToAMQPTable(options.Headers)
+		}
+
+		// If we have a TTL use it.
+		if options.Expiration != "" {
```

Comment:
This if statement is redundant and can be removed, message.Expiration is already an empty string